### PR TITLE
Temporary theme for Innovation Cup

### PR DESCRIPTION
**Pull request by julien-deramond**
> **Warning**
> DO NOT MERGE
> DO NOT REBASE ANYMORE: this PR must stay rebased on v5.2.1 (commit https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/dd8f0ff921caa5c8c19c8cc96148695c96ae2dda) until the end of time

### Description

Temporary theme for Innovation Cup

### Checklist

- [x] Create an "innovation cup" label that can be used here or when user will create issues
- [x] Lock the conversation so we don't have questions or feedback in this PR and _force_ the users to use issues
- [x] Make disappear all mentions of Orange and replace them by "Innovation Cup"
- [x] Make disappear all mentions of Orange and replace them by "Innovation Cup" in Licence + copyrights parts
- [x] Change the tutorials to use "boosted-innovation-cup" instead of "boosted"
- [x] Replace "boosted" by "boosted-innovation-cup" in packages names (BUT NOT in css and js)
- [x] Change all the v5.2.0 references in order to use the version for "boosted-innovation-cup" (Tried ` 5`, `v5`, `5.`, ` 6`, `v6`, `6.`, ` 4`, `v4`, `4.`, `version`, `release`, `\.[0-9]\.`, `[0-9]\.[0-9]`), `README`, `CONTRIBUTING`, templates (**missing  license + installation guide)**
- [ ] Protect this branch (so we will have to create PRs against it to make it evolve when it will be stabilized)
- [x] Remove Design Guidelines section
- [x] Remove all callouts refering to the Design System
- [x] Replace all colors (Note: keep a black header and play with blue colors instead of orange colors)  
- Remove Orange specific resources
  - [x] Remove Helvetica* fonts and let's have Arial by default
  - [x] Remove the logo 
  - [x] Remove other Orange images (don't forget favicons etc.)
  - [x] Remove Solaris icons
  - [x] In the header, replace the logo by a header with a title "Innovation Cup" (or title + icon) 
- [ ] Check all the docs
- [ ] A11y ⚠️ 
- [ ] Provide a NPM package (called "boosted-innovation-cup"?)
- Communication for the Innovation Cup
  - [x] Link the preview URL as the official documentation
  - [ ] Let the participants know that they can create issues ->create a template ?
- [x] Change the filename of dist files (the ones that are shared)
- [x] Search for OBS word and reformulate 
- [x] Search for Solaris word and links and reformulate / redirect to Bootstrap Icons
- [x] Check https://deploy-preview-1536--boosted.netlify.app/docs/5.2/customize/color/#core-colors where `$brand-orange` display some blue.
- [x] Check all components colors (e.g. list-group-action)
- [x] Change links from `blob/v*` to the appropriate one.

### Color mapping

| Scss vars | Aliases | Boosted color | Innovation Cup color |
|---|---|---|---|
| `$white` |  | `#fff`  | `#fff` |
| `$gray-100` |  | `#fafafa`  |   |
| `$gray-200` |  | `#f6f6f6`  |   |
| `$gray-300` |  | `#eee`  |   |
| `$gray-400` |  | `#ddd`  |   |
| `$gray-500` | `$light` | `#ccc`  |   |
| `$gray-600` |  | `#999`  |   |
| `$gray-700` |  | `#666`  |   |
| `$gray-800` |  | `#595959`  |   |
| `$gray-900` |  | `#333`  |   |
| `$black` | `$secondary`, `$dark`  | `#000`  |  `#b0bec5`  |
| `$orange` | `$brand-orange`, `$primary` | `#ff7900` | `#63a4ff`  |
| `$accessible-orange`| | `#f16e00`  | `#1976d2` |
| `$functional-green`| `$green`, `$success` | `#32c832`  | `#328532` |
| `$functional-blue`| `$blue`, `$info` | `#527edb`  |  `#4170d8`  |
| `$functional-yellow`|  `$yellow`, `$warning` | `#ffcc00`  |  `#ffcc00`  |
| `$functional-red`| `$red`, `$danger` | `#ab000d`  |  `#e32800`  |
| `$supporting-blue`| `$cyan` | `#4bb4e6`  |   |
| `$supporting-yellow`| | `#ffd200`  |   |
| `$supporting-green`| `$teal` | `#50be87`  |   |
| `$supporting-purple`| `$indigo`, `$purple` | `#a885d8`  |   |
| `$supporting-pink`| `$pink` | `#ffb4e6`  |   |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "orange-opensource/orange-boosted-bootstrap",
-  "description": "Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.",
+  "description": "Innovation Cup Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.",
```
Comment (isabellechanclou) [inline composer.json:3]: "Orange branded" in this sentence shouldn't be removed ?

File: config.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ module:
       target:                       static/docs/5.2/assets/js/lang
 
 params:
-  subtitle:                         "Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library…"
+  subtitle:                         "Innovation Cup Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library…"
```
Comment (isabellechanclou) [inline config.yml:57]: "Orange branded" in this sentence shouldn't be removed ?

File: nuget/boosted.nuspec [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <title>Boosted CSS</title>
     <authors>The Boosted Authors, Orange SA</authors>
     <owners>Orange SA</owners>
-    <description>Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.</description>
+    <description>Innovation Cup Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.</description>
```
Comment (isabellechanclou) [inline nuget/boosted.nuspec:10]: "Orange branded" in this sentence shouldn't be removed ?

File: composer.json (hunk 1 above)
Comment (julien-deramond) [reply composer.json:3]: Yes it should. There are still a loooot of modifications to do in the PR

File: site/content/docs/0.1/getting-started/download.md [hunk 4]
```diff
@@ -81,11 +53,3 @@ Boosted Innovation Cup's `package.json` contains some additional metadata under
 {{< callout info >}}
 {{< partial "callout-info-npm-starter.md" >}}
 {{< /callout >}}
-
-### yarn
```
Comment (julien-deramond) [inline site/content/docs/0.1/getting-started/download.md]: You must keep it. NPM packages are usable with npm and yarn

**Pull request closed without merging**